projects/helper/coreAssets.json

```diff
@@ -378,7 +378,9 @@
     "DAI": "0x90ca507a5d4458a4c6c6249d186b6dcb02a5bccd",
     "NCT": "0x02de4766c272abc10bc88c220d214a26960a7e92",
     "cMCO2": "0x32a9fe697a32135bfd313a6ac28792dae4d9979d",
-    "USDT": "0x617f3112bf5397d0467d315cc709ef968d9ba546"
+    "axlUSDT": "0x617f3112bf5397d0467d315cc709ef968d9ba546",
```

Comment:
please dont change existing mapping name, if some adapter is relying on this label, it will break, you can add the new USDT as "USDT_1"